src/System.Runtime.Extensions/tests/System/BitConverter.cs

```diff
@@ -207,7 +209,20 @@ public static void RoundtripString()
             Assert.Equal("12-34-56-78-9A", BitConverter.ToString(bytes));
             Assert.Equal("56-78-9A", BitConverter.ToString(bytes, 2));
             Assert.Equal("56", BitConverter.ToString(bytes, 2, 1));
-            Assert.Equal(String.Empty, BitConverter.ToString(new byte[0]));
+
+            Assert.Same(string.Empty, BitConverter.ToString(new byte[0]));
+            Assert.Same(string.Empty, BitConverter.ToString(new byte[3], 1, 0));
```

Comment:
When tests are open-source, they become documentation of a sort in themselves, in some ways better than docs (if they're incorrect, they fail) but in others worse in that something like this can be seen as a promise even when not intended as such. A comment of "not promised by spec" or "consider impact before changing this behaviour" would remove that downside.